cliquet/tests/test_utils.py

```diff
@@ -101,3 +105,50 @@ def test_current_service_returns_none_for_unexisting_patterns(self):
         request.registry.cornice_services = {}
 
         self.assertEqual(current_service(request), None)
+
+
+class EncodeHeaderTest(unittest.TestCase):
+
+    def test_giving_string_return_a_string(self):
```

Comment:
nit: in other test cases, the formulation is something like ``test_returns_blah_if_foo``